src/config/config.ts

```diff
@@ -0,0 +1,360 @@
+        env: "OUTGOING_QUEUE_URL",
+        format: "required-string",
+        default: null,
+      },
+      siteLaunchQueueUrl: {
+        doc: "URL of the site launch SQS queue",
+        env: "SITE_LAUNCH_QUEUE_URL",
+        format: "required-string",
+        default: null,
+      },
+    },
+  },
+  github: {
+    orgName: {
+      doc: "GitHub organization that owns all site repositories",
+      env: "GITHUB_ORG_NAME",
+      format: "required-string",
+      default: "isomerpages",
+    },
+    buildOrgName: {
+      doc: "GitHub organization that owns the build repository",
+      env: "GITHUB_BUILD_ORG_NAME",
+      format: "required-string",
+      default: "opengovsg",
+    },
+    buildRepo: {
+      doc: "Name of the build GitHub repository",
+      env: "GITHUB_BUILD_REPO_NAME",
+      format: "required-string",
+      default: "isomer-build",
+    },
+    clientId: {
+      doc: "GitHub OAuth app Client ID",
+      env: "CLIENT_ID",
+      format: "required-string",
+      default: null,
+    },
+    clientSecret: {
+      doc: "GitHub OAuth app Client secret",
+      env: "CLIENT_SECRET",
+      sensitive: true,
+      format: "required-string",
+      default: null,
+    },
+    redirectUri: {
+      doc: "URL to redirect to after authentication with GitHub",
+      env: "REDIRECT_URI",
+      format: "required-string",
+      default: null,
+    },
+    branchRef: {
+      doc: "Git branch to use for saving modifications to site",
+      env: "BRANCH_REF",
+      format: "required-string",
+      default: "staging",
+    },
+    systemToken: {
+      doc: "GitHub access token to create repo",
+      env: "SYSTEM_GITHUB_TOKEN",
+      sensitive: true,
+      format: "required-string",
+      default: null,
+    },
+  },
+  dataDog: {
+    env: {
+      doc: "The DataDog environment",
+      format: ["production", "development", "local"],
+      env: "DD_ENV",
+      default: "local",
+    },
+    service: {
+      doc: "The DataDog service",
+      env: "DD_SERVICE",
+      format: "required-string",
+      default: null,
+    },
+    tags: {
+      doc: "The DataDog tags",
+      env: "DD_TAGS",
+      format: "required-string",
+      default: null,
+    },
+  },
+  formSg: {
+    siteCreateFormKey: {
+      doc: "FormSG API key for site creation form",
+      env: "SITE_CREATE_FORM_KEY",
+      sensitive: true,
+      format: "required-string",
+      default: null,
+    },
+  },
+  postman: {
+    apiKey: {
+      doc: "Postman API key",
+      env: "POSTMAN_API_KEY",
+      sensitive: true,
+      format: "required-string",
+      default: "",
+    },
+    smsCredName: {
+      doc: "Postman SMS credential name",
+      env: "POSTMAN_SMS_CRED_NAME",
+      format: "required-string",
+      default: "",
+    },
+  },
+  cypress: {
+    e2eTestRepo: {
+      doc: "Name of the e2e test GitHub repository",
+      env: "E2E_TEST_REPO",
+      format: "required-string",
+      default: null,
```

Comment:
can we just give the actual e2e repo name here cos idt it's sensitive

Reply:
@seaerchin updated to `"e2e-test-repo"`